demo/main.cc

```diff
@@ -17,21 +17,21 @@
 #include <thread>
 
 int main(int argc, char *argv[]) {
-    try {
-        chainmain_process();   // chain-main
-        test_chainmain_nft();  // chainmain nft tests
-        test_login();          // decentralized login
-        cronos_process();      // cronos
-        test_cronos_testnet(); // cronos testnet
-    } catch (const rust::cxxbridge1::Error &e) {
-        // Use `Assertion failed`, the same as `assert` function
-        std::cout << "Assertion failed: " << e.what() << std::endl;
-    }
+  try {
+    chainmain_process();   // chain-main
+    test_chainmain_nft();  // chainmain nft tests
+    test_login();          // decentralized login
+    cronos_process();      // cronos
+    test_cronos_testnet(); // cronos testnet
+  } catch (const rust::cxxbridge1::Error &e) {
+    // Use `Assertion failed`, the same as `assert` function
+    std::cout << "Assertion failed: " << e.what() << std::endl;
+  }
```

Comment:
the indentations do not align with other cc files.

Reply:
keep default 4 spaces?

Reply:
ok

Reply:
change default tab to 4 spaces
```
find . -name "*.h" -o -name "*.cpp" -o -name "*.cc" -o -name "*.cs" | xargs clang-format -i -style="{SortIncludes: Never, IndentWidth: 4}"
```

Reply:
to be consistent, applied to all c++ source code

Reply:
> to be consistent, applied to all c++ source code

can this be done in a separate PR, so the the diff on this one is smaller?

Reply:
yes, i'll submit as another pr